sirepo/template/template_common.py

```diff
@@ -346,8 +346,8 @@ def sim_frame_dispatch(frame_args):
         ),
     )
     t = sirepo.template.import_module(frame_args.simulationType)
-    o = getattr(t, 'sim_frame', None) \
-        or getattr(t, 'sim_frame_' + frame_args.frameReport)
+    o = getattr(t, 'sim_frame_' + frame_args.frameReport, None) \
```

Comment:
@moellep I want to call you attention to this. I changed the logic (try to use sim_frame_* before using generic sim_frame). The only existing code it should impact is Opal (that was the only code with sim_frame and sim_frame_*) which I changed. I tested all Opal examples and it seems fine but double check my logic if you don't mind.